src/monodroid/jni/monodroid-glue.c

```diff
@@ -3361,13 +3361,16 @@ set_profile_options (JNIEnv *env)
 	if (!output) {
 		const char* col = strchr (value, ':');
 		char *ovalue;
+		char *filename = monodroid_strdup_printf ("profile.%s",
+					strncmp (value, "log", 3) ? "aot" : "mlpd");
```

Comment:
> It is now set to `profile.aot` for the aot profiler and `.mlpd` for the log profiler.

From this code, we see that if the `debug.mono.log` value *starts with* `log`, we use `.mlpd` as the extension; Otherwise, we use `.aot`.

What value should be used instead of `log` to get the AOT profiler? `aot`? Something else?

Please explicitly check for known values instead of treating anything other than `log` as AOT.

Reply:
Currently only the log and aot work for `mono.debug.profile` in XA. This part of code is not performance critical, so I guess it indeed makes sense to check for both.

I think we might actually use `.mldp` for log profiler and profiler module name for others, so that it might not need change in future, if we make it possible to use another mono profiler module in XA.

I will update it.